local/components/kostyanvi/news/class.php

```diff
@@ -0,0 +1,66 @@
+<? if (!defined("B_PROLOG_INCLUDED") || B_PROLOG_INCLUDED !== true) {
+	die();
+}
+
+use Bitrix\Main\Engine\Contract\Controllerable;
+use Bitrix\Main\Web\Json;
+
+class CNewsComponent extends CBitrixComponent implements Controllerable
+{
+	public function configureActions()
+	{
+		return [];
+	}
+
+	public function executeComponent()
+	{
+		$this->includeComponentTemplate();
+	}
+
+	public function getNewsByIDAction()
+	{
+		if (\Bitrix\Main\Loader::includeModule('iblock')) {
+			$newsDetailInfo = [];
+			$request = \Bitrix\Main\Context::getCurrent()->getRequest();
+
+			$detailEl = \Bitrix\Iblock\ElementTable::getById($request->get('id'));
+
+			$newsDetailInfo = $detailEl->fetch();
```

Comment:
Тоже лишняя переменная:

```suggestion
			$newsDetailInfo = \Bitrix\Iblock\ElementTable::getById($request->get('id'))->fetch();
```